client_plugin/drivers/vfile/kvstore/etcdops/etcdops.go

```diff
@@ -324,8 +420,8 @@ func (e *EtcdKVS) checkLocalEtcd() error {
 						"error": err},
 				).Warningf("Failed to get ETCD client, retry before timeout ")
 			} else {
+				e.watcher = cli
 				go e.etcdWatcher(cli)
-				go e.serviceAndVolumeGC(cli)
```

Comment:
Why here we don't need to call e.serviceAndVolumeGC?

Reply:
serviceAndVolumeGC is renamed to etcdHelper with the role check function inside now.
It's now moved to outside of checkLocalEtcd and after joinETCD/startETCD, so the joinETCD function can be re-used by etcdHelper itself.